# XML output to test/ref/exampleXmlDefault.xml
# Started on 02/07/15 15:30:07
# Starting class: TestTiti
# Starting test: TestTiti.test1_withFailure
TestTiti:setUp
some stuff test 1
# Failure: example_with_luaunit.lua:99: expected: 2, actual: 1
TestTiti:tearDown
# Starting test: TestTiti.test2_withFailure
TestTiti:setUp
some stuff test 2
# Failure: example_with_luaunit.lua:108: expected: "bof"
actual: "hop"
TestTiti:tearDown
# Starting test: TestTiti.test3
TestTiti:setUp
some stuff test 3
TestTiti:tearDown
# Starting class: TestToto
# Starting test: TestToto.test1_withFailure
some stuff test 1
# Failure: example_with_luaunit.lua:19: expected: 2, actual: 1
# Starting test: TestToto.test2_withFailure
some stuff test 2
# Failure: example_with_luaunit.lua:28: expected: "bof"
actual: "hop"
# Starting test: TestToto.test3
some stuff test 3
# Starting test: TestToto.test4
some stuff test 4
# Failure: example_with_luaunit.lua:41: Received the not expected value: 1
# Starting test: TestToto.test5
some stuff test 5
# Failure: example_with_luaunit.lua:47: expected: false, actual: 1
# Starting test: TestToto.test6
some stuff test 6
# Failure: example_with_luaunit.lua:52: expected: true, actual: false
# Starting test: TestToto.test7
# Failure: example_with_luaunit.lua:58: expected: {1, 2, three=3}
actual: {1, 2}
# Starting test: TestToto.test8a
# Failure: example_with_luaunit.lua:76: Bouhouhoum error!
# Starting test: TestToto.test8b
# Failure: example_with_luaunit.lua:76: Bouhouhoum error!
# Starting test: test1_withFailure
# Failure: example_with_luaunit.lua:124: assertion failed!
# Starting test: test2_withFailure
# Failure: example_with_luaunit.lua:130: assertion failed!
# Starting test: test3
# Ran 15 tests in 0.000 seconds, 3 successes, 12 failures
